alvr/session/src/settings.rs

```diff
@@ -479,6 +479,11 @@ pub struct VideoConfig {
 
     pub clientside_foveation: Switch<ClientsideFoveationConfig>,
 
+    #[schema(strings(
+        help = "Attempts to use a software decoder on the device. Slow, but may work around broken codecs."
+    ))]
+    pub force_software_decoder: bool,
```

Comment:
Move this just above `mediacodec_extra_options`

Reply:
Done as of [d9a8731](https://github.com/alvr-org/ALVR/pull/1933/commits/d9a8731457c6bf32e5c7dd6fd5490a68ae2146f2).